python/cudf/cudf/pandas/scripts/conftest-patch.py

```diff
@@ -437,4 +437,49 @@ def pytest_runtest_makereport(item, call):
             rep.wasxfail = f"xfail due to known error: {error_message[:50]}"
 
 
+SKIP_TEST_PATTERNS = [
+    "test_reduce_series_numeric[uint8-kurt-False]",
+    "test_reduce_series_numeric[uint8-skew-False]",
+    "test_reduce_series_numeric[uint16-kurt-False]",
+    "test_reduce_series_numeric[uint16-skew-False]",
+    "test_reduce_series_numeric[uint32-kurt-False]",
+    "test_reduce_series_numeric[uint32-skew-False]",
+    "test_reduce_series_numeric[uint64-kurt-False]",
+    "test_reduce_series_numeric[uint64-skew-False]",
+    "test_reduce_series_numeric[int8-kurt-False]",
+    "test_reduce_series_numeric[int8-skew-False]",
+    "test_reduce_series_numeric[int16-kurt-False]",
+    "test_reduce_series_numeric[int16-skew-False]",
+    "test_reduce_series_numeric[int32-kurt-False]",
+    "test_reduce_series_numeric[int32-skew-False]",
+    "test_reduce_series_numeric[int64-kurt-False]",
+    "test_reduce_series_numeric[int64-skew-False]",
+    "test_reduce_series_numeric[float-kurt-False]",
+    "test_reduce_series_numeric[float-skew-False]",
+    "test_reduce_series_numeric[double-kurt-True]",
+    "test_reduce_series_numeric[double-kurt-False]",
+    "test_reduce_series_numeric[double-skew-True]",
+    "test_reduce_series_numeric[double-skew-False]",
+    "test_reduce_series_numeric[decimal128(7, 3)-kurt-False]",
+    "test_reduce_series_numeric[decimal128(7, 3)-skew-False]",
+    "test_reduce_series_numeric[bool-std-True]",
+    "test_reduce_series_numeric[bool-std-False]",
+    "test_reduce_series_numeric[bool-var-True]",
+    "test_reduce_series_numeric[bool-var-False]",
+    "test_reduce_series_numeric[bool-median-True]",
+    "test_reduce_series_numeric[bool-median-False]",
+    "test_reduce_series_numeric[bool-kurt-False]",
+    "test_reduce_series_numeric[bool-skew-False]",
+]
+
+
+@pytest.hookimpl(tryfirst=True)
+def pytest_runtest_setup(item):
+    """Skip tests that match patterns in SKIP_TEST_PATTERNS"""
+    test_name = item.name
+    for pattern in SKIP_TEST_PATTERNS:
+        if test_name == pattern:
+            pytest.skip(f"Skipping known problematic test: {test_name}")
```

Comment:
Why are these problematic? Can they just not be skipped how other tests are skipped in `python/cudf/cudf/pandas/scripts/run-pandas-tests.sh`?

Reply:
These are tests that xfail(strict=True) in pandas but pass in `cudf.pandas`, I tried adding to `run-pandas-tests.sh` but the list seems to be at it's limits that pytest is crashing if I do it there. The plan is to move all the `.sh` skip entries into here - but as a separate PR.

Reply:
Gotcha, thanks. Yeah I have a section in https://github.com/rapidsai/cudf/pull/18872 to put these XPASSing test when we move towards not skipping anymore

Reply:
Could you leave a comment here that these test are xpassing so we don't forget?

Reply:
Yup, I plan to merge your PR before merging mine. I'll then update this script.